src/main/resources/db/changelog/db.changelog-RDM-9696.xml

```diff
@@ -0,0 +1,20 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <changeSet id="rdm-9696" runInTransaction="false" author="suman.sreenivasan@hmcts.net">
+        <sql dbms="postgresql"
+             endDelimiter="\nGO"
+             splitStatements="true"
+             stripComments="true">
+            CREATE INDEX CONCURRENTLY idx_case_data_languagePreferenceWelsh ON public.case_data USING btree (btrim(upper((data #>> '{languagePreferenceWelsh}'::text[]))));
```

Comment:
each index should be created if it doesn't exist.

Reply:
changed the query.